BUILDING.md

```diff
@@ -97,58 +97,81 @@ Node.js does not support a platform version if a vendor has expired support
+| macOS            | x64              | >= 10.13                        |                                   |
+
+<em id="tier1-fn1">1</em>: GCC 6 is not provided on the base platform. Users will
   need the
   [Toolchain test builds PPA](https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/test?field.series_filter=xenial)
   or similar to source a newer compiler.
 
-<em id="fn2">2</em>: GCC 6 is not provided on the base platform. Users will
+<em id="tier1-fn2">2</em>: GCC 6 is not provided on the base platform. Users will
   need the
   [devtoolset-6](https://www.softwarecollections.org/en/scls/rhscl/devtoolset-6/)
   or later to source a newer compiler.
 
-<em id="fn3">3</em>: Older kernel versions may work for ARM64. However the
+<em id="tier1-fn3">3</em>: Older kernel versions may work for ARM64. However the
   Node.js test infrastructure only tests >= 4.5.
 
-<em id="fn4">4</em>: On Windows, running Node.js in Windows terminal emulators
+<em id="tier1-fn4">4</em>: On Windows, running Node.js in Windows terminal emulators
   like `mintty` requires the usage of [winpty](https://github.com/rprichard/winpty)
   for the tty channels to work (e.g. `winpty node.exe script.js`).
   In "Git bash" if you call the node shell alias (`node` without the `.exe`
   extension), `winpty` is used automatically.
 
-<em id="fn5">5</em>: The Windows Subsystem for Linux (WSL) is not
+<em id="tier1-fn5">5</em>: The Windows Subsystem for Linux (WSL) is not
   supported, but the GNU/Linux build process and binaries should work. The
   community will only address issues that reproduce on native GNU/Linux
   systems. Issues that only reproduce on WSL should be reported in the
   [WSL issue tracker](https://github.com/Microsoft/WSL/issues). Running the
   Windows binary (`node.exe`) in WSL is not recommended. It will not work
   without workarounds such as stdio redirection.
 
-<em id="fn6">6</em>: Running Node.js on x86 Windows should work and binaries
+<em id="tier1-fn6">6</em>: Running Node.js on x86 Windows should work and binaries
 are provided. However, tests in our infrastructure only run on WoW64.
 Furthermore, compiling on x86 Windows is Experimental and
 may not be possible.
 
-<em id="fn7">7</em>: The default FreeBSD 12.0 compiler is Clang 6.0.1, but
+
+#### Tier 2
+
+| Operating System | Architectures    | Versions                        | Notes                             |
+| ---------------- | ---------------- | ------------------------------- | --------------------------------- |
+| GNU/Linux        | ppc64le >=power8 | kernel >= 3.10.0, glibc >= 2.17 | e.g. Ubuntu 16.04 <sup>[1](#tier2-fn1)</sup>, EL 7  <sup>[2](#tier2-fn2)</sup> |
+| GNU/Linux        | s390x            | kernel >= 3.10.0, glibc >= 2.17 | e.g. EL 7 <sup>[2](#tier2-fn2)</sup>              |
+| Windows          | arm64            | >= Windows 10                   | Tier 2 (compiling) / Experimental (running) |
```

Comment:
I think we should decide what these tier tables represent, either for compilation or execution. I personally think it should indicate support for execution, in which case this entry would need to be moved to the 'Experimental' table below.

Reply:
Or maybe duplicate these split support entries and just note which type in the comments?

Reply:
I found this surprising as well, as I'd not noticed that we'd had that. I kind of agree that it should be for execution.

Reply:
I'd agree that in this PR we should move to experimental versus Tier 2 as I think that could lead to people thinking it is support at a higher level than it is.

Reply:
I'm fine with moving it to experimental, the last column can explain more clearly what the status is if needed. For practical purposes, the fact that it compiles isn't really very interesting. People want to know if they can _use_ it.